pootle/checks.py

```diff
@@ -262,6 +262,50 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W015",
                 ))
 
+    if settings.POOTLE_TM_SERVER:
+        tm_indexes = []
+
+        if 'local' not in settings.POOTLE_TM_SERVER:
+            errors.append(checks.Critical(
+                _("POOTLE_TM_SERVER['local'] is missing.", server),
+                hint=_("Set POOTLE_TM_SERVER['local']."),
+                id="pootle.C008",
+            ))
+
+        for server in settings.POOTLE_TM_SERVER:
+            if 'INDEX_NAME' not in settings.POOTLE_TM_SERVER[server]:
+                errors.append(checks.Critical(
```

Comment:
Removing this check would require to adjust the code in other places in order to ensure we have a `INDEX_NAME`. I prefer to keep that check just here. And this is a critical error since otherwise the TM is unusable.